server/src/index.js

```diff
@@ -0,0 +1,45 @@
+require("dotenv").config();
+require("reflect-metadata");
+
+const express = require("express");
+const { ApolloServer } = require("apollo-server-express");
+const cors = require("cors");
+const { createServer } = require("http");
+const cookieParser = require("cookie-parser");
+
+const sequelize = require("./db/sequelize");
+const resolvers = require("./graphql/resolvers/index");
+const typeDefs = require("./graphql/schemas/index");
+
+const startServer = async () => {
+  const server = new ApolloServer({
+    typeDefs,
+    resolvers,
+    context: ({ req, res }) => ({ req, res }),
+  });
+  console.log(process.env.SQL_CONNECTION_URL);
```

Comment:
does this console log serve any purpose?

Reply:
no - removed and rebased